uvloop/includes/stdlib.pxi

```diff
@@ -134,6 +134,7 @@ cdef int ssl_SSL_ERROR_WANT_WRITE = ssl.SSL_ERROR_WANT_WRITE
 cdef int ssl_SSL_ERROR_SYSCALL = ssl.SSL_ERROR_SYSCALL
 
 cdef uint64_t MAIN_THREAD_ID = <uint64_t><int64_t>threading.main_thread().ident
+cdef cc_Thread = threading.Thread
```

Comment:
Please rename to `threading_Thread`

Reply:
Oh, I just realized that `cc` was short for *concurrent* in the context of `cc_ThreadPoolExecutor`, haha. Didn't catch that earlier since I usually see `concurrent.futures` abbreviated as `cf`.